pkg/client/dialer.go

```diff
@@ -54,26 +55,33 @@ func Dial(ctx context.Context, vat vat.Network, a Addr) (*Node, error) {
 // to abide by this rule may cause Node's underlying capabilities
 // to fail.
 func (d Dialer) Dial(ctx context.Context) (*Node, error) {
-	conn, err := d.join(ctx)
+	n := &Node{vat: d.Vat, conns: make([]*rpc.Conn, 0)}
```

Comment:
I wasn't sure how you wanted to manage all the connections, so I simply created a slice of connections.

Reply:
Agreed.  This is good enough for now.  We're going to have to put some deliberate thought into how to manage auth/conns/caps/etc very soon, anyway.